cinema/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    duration = models.IntegerField()
+    description = models.TextField(null=True, blank=True)
```

Comment:
In task description isn't specified that movie description could be null
Let's fix that and suggest that we always want for our customers to be able to read about the movie, for example. It would be strange not to be able to see at least some description of the movie when buying tickets, wouldn't it?